uploader.v2.json

```diff
@@ -1642,6 +1642,11 @@
             ],
             "symbolic_name": "etcd_conf"
         },
+        {
+            "file": "/var/log/libvirt/libvirtd.log",
+            "pattern": [],
+            "symbolic_name": "libvirtd_log"
+        },
```

Comment:
Same as above.

Reply:
This is removed and will be added as a separate specs.